src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -74,6 +74,14 @@ enum RequisitionStatus {
         [CREATED, EDITING, VERIFYING, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
+    static listOutboundOptionsWhenSupportingRequestApproval() {
+        [CREATED, EDITING, APPROVED, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
+    }
+
+    static listRequisitionOptions() {
```

Comment:
I don't know if it's a good naming here - what do you mean by "Requisition" here?

Reply:
I changed the naming here, but I am still open to better propositions